doc/authors.rst

```diff
@@ -0,0 +1,19 @@
+.. _Mainak Jas: https://perso.telecom-paristech.fr/mjas/
+.. _Teon Brooks: https://teonbrooks.github.io/
+.. _Chris Holdgraf: https://bids.berkeley.edu/people/chris-holdgraf
+.. _Matt Sanderson: https://github.com/monkeyman192
+.. _Stefan Appelhoff: http://stefanappelhoff.com/
+.. _Romain Quentin: https://github.com/romquentin
+.. _Dominik Welke: https://github.com/dominikwelke
+.. _Maximilien Chaumon: https://github.com/dnacombo
+.. _Ezequiel Mikulan: https://github.com/ezemikulan
+.. _Marijn van Vliet: https://github.com/wmvanvliet
+.. _Alex Rockhill: http://github.com/alexrockhill
+.. _Sophie Herbst: http://github.com/SophieHerbst
+.. _Adam Li: https://github.com/adam2392
+.. _Fu-Te Wong: https://github.com/zuxfoucault
+.. _Richard Höchenberger: https://github.com/hoechenberger
```

Comment:
why is your name gray here @hoechenberger or is it only in my UI?

![grafik](https://user-images.githubusercontent.com/9084751/96504416-f0b6d980-1254-11eb-8c7b-51e53aacd7b6.png)

Reply:
I have no idea! Hopefully it's a good thing LOL